app/javascript/packs/modal.js

```diff
@@ -4,12 +4,12 @@ document.addEventListener('turbolinks:load', function(){
   
   function openModal(event) {
     var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
```

Comment:
The "id" attribute must be unique, don't need pointer to tag "div", can replace this line with
    `var modal = document.querySelector("#" + event.currentTarget.id);`

Reply:
The id of the button that calls the modal and the id of the modal, we link them so that it will work in this way, that's why we add the div, to identify that the modal is being talked about.

Reply:
Need to check if the error is here or when calling the function, [check the documentation](https://guides.rubyonrails.org/working_with_javascript_in_rails.htmll)